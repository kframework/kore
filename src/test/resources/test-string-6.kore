[hook{}(\dv{Hook{}}("\u00A9"))]

module TEST-STRING-6

endmodule []
