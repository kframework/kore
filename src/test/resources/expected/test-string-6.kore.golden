Definition
    { definitionAttributes =
        Attributes 
            [ Fix (UnifiedPattern (UnifiedObject (Rotate31 (ApplicationPattern Application
                { applicationSymbolOrAlias =
                    SymbolOrAlias
                        { symbolOrAliasConstructor = (Id "hook" AstLocationNone) :: Id Object
                        , symbolOrAliasParams = []
                        }
                , applicationChildren =
                    [ Fix (UnifiedPattern (UnifiedObject (Rotate31 (DomainValuePattern DomainValue
                        { domainValueSort =
                            SortActualSort SortActual
                                { sortActualName = (Id "Hook" AstLocationNone) :: Id Object
                                , sortActualSorts = []
                                }
                        , domainValueChild =
                            Fix (UnifiedPattern (UnifiedMeta (Rotate31 (StringLiteralPattern (StringLiteral "\251")))))
                        }))))
                    ]
                }))))
            ]
    , definitionModules =
        [ Module
            { moduleName = ModuleName "TEST-STRING-6"
            , moduleSentences = []
            , moduleAttributes = Attributes []
            }
        ]
    }